src/lib/components/multiDropzone/types.ts

```diff
@@ -0,0 +1,34 @@
+import { Accept } from "react-dropzone";
+
+export type UploadStatus = 'UPLOADING' | 'COMPLETE' | 'ERROR';
+
+export const DOCUMENT_FILES = ['doc', 'docx', 'pdf'];
+export const IMAGE_FILES = ['heic', 'bmp', 'jpeg', 'jpg', 'png'];
+
+export const FILE_TYPES = [...DOCUMENT_FILES, ...IMAGE_FILES];
+export type FileType = typeof FILE_TYPES[number];
+
+export interface UploadedFile {
+  id: string;
+  name: string;
+  type: FileType | string;
```

Comment:
Is there a reason to leave `string` here? I feel like doing so removes the safety from the interface.
What do you think?

Reply:
Fair point. I would keep it strict and add more types as needed, but maybe we can asses this in another PR just to be able to merge this one.

Reply:
I'll create an issue to revisit this, we can probably add some proper way of checking types and mime types